### ISSUE TITLE
[Feature Request] Non Rotating Variants / Rotation Protection

### ISSUE THREAD

**Issue by ghost** [MongoTheElder/PITG#23]
I like using these as nicer-looking dividers between people and images/text in some areas where a claim isn't really possible, in these situations I am left with two options as far as stopping people from rotating them and editing the text/images:

1. Disable the glazing tool outright (Nobody can rotate, feelsbadman.) or: 
2. Hope nobody takes their own glaze tool to it / rotates it.

Unbreakable is great (Albeit I use claims, in some instances I can't and this helps.) but a way to prevent others from rotating your own edge-aligned glass pieces with their glazing tools while still having rotation would be a big, big help.

Possible options to do this on your end:

- Make Un-rotatable versions of all the edge-aligned glass. (Probably easiest.)
- Make Edge-Aligned Glass have a config option/ability to only be rotatable by owner/placer.
- Have a tool to change/designate pieces to not be rotatable.

 
**_tl:dr; I'de really like to have non-rotatable versions/rotation protection for less griefability/flexibility. I want and think this feature would really put the cherry on top so to speak._**

**Comment by MongoTheElder** [comment]
I can make setting 'unbreakable' to include 'non-rotatable' (that makes sense anyway). Not sure adding another mode is workable without breaking previous builds. I don't have a way to track block "ownership" since these are not block entities and there is no clean way to implement block specific data without making these block entities which I probably won't do (for various performance reasons)

**Comment by MongoTheElder** [comment]
Implemented in 1.19.1
